migration/migrator/data/course_tables.sql

```diff
@@ -201,6 +201,8 @@ CREATE TABLE public.electronic_gradeable (
     eg_peer_blind integer DEFAULT 3,
     eg_grade_inquiry_start_date timestamp(6) with time zone NOT NULL,
     eg_hidden_files character varying(1024),
+    eg_depends_on character varying(255) DEFAULT NULL::character varying,
```

Comment:
Looks like `eg_depends_on` should only be set to null or a valid `g_id`. As such, we should add a [foreign key](https://www.postgresqltutorial.com/postgresql-foreign-key/) constraint to ensure data consistency, as well as handling the case where a gradeable is deleted where another gradeable depends on it.